cfme/networks/cloud_tenant.py

```diff
@@ -0,0 +1,72 @@
+import attr
+
+from cfme.cloud.tenant import TenantDetailsView
+from cfme.common import Taggable
+from cfme.modeling.base import BaseCollection, BaseEntity, parent_of_type
+from cfme.networks import ValidateStatsMixin
+from cfme.utils import version
+from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+
+
+@attr.s
+class CloudTenant(Taggable, BaseEntity, ValidateStatsMixin):
```

Comment:
This still appears to be a duplication of the Tenant/TenantCollection classes that already exist in `cfme/cloud/tenant.py`, or at least poorly named for its intent.

The tenants that this is apparently tied to, when you're on the page what vertical navigation is it under? Are they the same as the cloud provider tenants?

If so, then the convenience methods implemented here could be implemented elsewhere, or a mechanism with less boilerplate could be used.  There could be a single navigation destination for a Tenant details page, that could take into account any filters applied to its parent collection.